config/test.exs

```diff
@@ -20,3 +20,6 @@ config :drag_time_api, DragTimeApiWeb.Endpoint,
 
 # Print only warnings and errors during test
 config :logger, level: :warn
+
+# Bcrypt configuration 
+config :bcrypt_elixir, :log_rounds, 4
```

Comment:
Nice

Reply:
I should actually take this off now since I used Phoenix's internal token tool instead of bcrypt